app/assets/javascripts/gallery.js.erb

```diff
@@ -1267,15 +1256,32 @@ $(document).ready(function(){
   /* Hack to fix space key so it can appropriately toggle hidden table cells. */
   $('.dataTable > tbody > tr > td:first-child').keypress(function(e){
     var keycode = (e.keyCode ? event.keyCode : event.which);
-    if(keycode == '32'){
+    if (keycode == '32'){
       e.preventDefault();
       $(this).click();
     }
   });
 
+  /* ===== Tooltips & Sparkline ===== */
```

Comment:
You moved this but forgot to remove the original
https://github.com/nbgallery/nbgallery/blob/death-to-bootbox/app/assets/javascripts/tooltips.js